rclpy/rclpy/subscription.py

```diff
@@ -72,9 +72,3 @@ def handle(self):
 
     def destroy(self):
         self.handle.destroy()
```

Comment:
Should we set the handle to `None` after it has been destroyed?

Reply:
It's meant to be kept around. Trying to use the handle after this raises `InvalidHandle`.